Add HTML example for <details>

Regaddi opened the pull request:
No github issue created so far.
I added some more styles to the element to make clear that it is a fully customizable element.

wbamberg commented on live-examples/html-examples/interactive-elements/css/details.css:
```diff
@@ -0,0 +1,41 @@
+details {
+    border: 1px solid #aaaaaa;
```
We use short form hex codes when we can, so this should be `#aaa`.

Regaddi commented:
@wbamberg Thank you very much for the detailed review, I really appreciate your input!

I adjusted styles and content accordingly. Using the `::-webkit-details-marker` was truly insufficient. I removed it to engage simplicity in the example.
